src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -81,6 +81,16 @@ internal class TokenAcquisition : ITokenAcquisition, ITokenAcquisitionInternal
             OidcConstants.ScopeOfflineAccess,
         };
 
+        /// <summary>
+        /// meta-tenant identifiers which are not allowed in client credentials.
+        /// </summary>
+        private readonly string[] unwantedTenantIdentifiersInClientCredentials = new string[]
+        {
```

Comment:
nit: start w/`_` how about `_metaTenantIdentifiers` or `_invalidTenantIdentifiersInClientCredentials` or ...?
Also, we have a const file, so I would put these there as const values, we might use them other places and those can be replaced w/the constant value too.

Reply:
Thanks for the tip.
Addressed!